### ISSUE TITLE
Image generation of different ipc

### ISSUE THREAD

**Issue by zero-0425** [vimar-gu/MinimaxDiffusion#16]
Hello, I am a little confused. when use "python sample.py --model DiT-XL/2 --image-size 256 --ckpt ../logs/run-0/000-DiT-XL-2-minimax/checkpoints/0012000.pt \
    --save-dir ../results/dit-distillation/imagenet-10-1000-minimax --spec woof" to generate images of ipc=10;  do you need to change the --num-samples from default 100 to 10. Or keep the default value 100 and randomly select the 10 images from 100 generated images via "python train.py -d imagenet --imagenet_dir /data/datasets/ImageNet/ \
    -n resnet_ap --nclass 10 --norm_type instance --ipc 10 --tag test --slct_type random --spec woof"

**Comment by vimar-gu** [comment]
Sorry for the late reply. You can select either of these two options. As the images are generated from random noise, the choice won't have a significant influence on the final performance. In my implementation, for convenience, I first generated 100 images, and randomly selected different IPCs from the set for validation.

**Comment by zero-0425** [comment]
Thanks for your reply. Under the IPC=100 setting, the selected images are expected to be both diverse and representative according to the loss constraint, meaning they should exhibit significant variation across samples. I am confused that randomly selecting different images from IPC100 won't have a significant influence on the final performance.

**Comment by vimar-gu** [comment]
The loss constraint is applied during training to encourage diverse generation results. However, there is no explicit constraint for inference. During inference, all the samples are independently generated from random noise. We didn't apply any trick to control the diversity or representativeness of the noise.

**Comment by zero-0425** [comment]
Thank you very much. I see, so even though a loss constraint is added during training, the generated images may not be diverse because of the random noise.

**Comment by vimar-gu** [comment]
Actually it is the opposite way. We want to convey that after fine-tuning, even if the images are generated from random noise, they still have better diversity than the baseline (DiT).
